webpack.config.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+    entry: ["./public/scripts/history.js", "./public/scripts/confetti.js", "./public/scripts/main.js","./public/scripts/keymaster.js" ],
+    target: 'node',
+    output: {
+        path: __dirname + "/dist",
+        filename: "bundle.js",
+        libraryTarget: 'var',
+        library: 'ui'
+    },
+    
+    mode: "production",
```

Comment:
For now, I think the mode would be better as development so developers can more easily see their errors. Production mode minifies the JS so it's hard to track down. When we get to building the prod config later, we can have it change but for now always development will work.